run.sh

```diff
@@ -98,6 +103,11 @@ if [ -z "$GALAXY_CONFIG_FILE" ]; then
     export GALAXY_CONFIG_FILE
 fi
 
+if [ $INITIALIZE_TOOL_DEPENDENCIES -eq 1 ]; then
+    # Install Conda environment if needed.
+    python ./scripts/manage_tool_dependencies.py -c "$GALAXY_CONFIG_FILE" init_if_needed
```

Comment:
You're right, `common_startup.sh` is not the right place for the reason you mentioned. But I was concerned about servers started through e.g. supervisor by invoking directly `uwsgi` or `./scripts/paster.py` .